src/main/java/com/onelogin/AccountSettings.java

```diff
@@ -1,7 +1,16 @@
 package com.onelogin;
 
+import java.io.ByteArrayInputStream;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+
+import org.apache.commons.codec.binary.Base64;
+
+
 public class AccountSettings {
 	private String certificate;
+	private Certificate cert;
```

Comment:
Please, rename cert by idp_cert.

Soon you will need to support certs of the SP and you will need to difference them.